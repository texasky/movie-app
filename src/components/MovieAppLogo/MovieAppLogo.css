.movie-app-logo {
    display: flex;
    align-items: center;
}

.movie-app-logo img {
    margin-right: 30px;
}

@media screen and (max-width: 700px) {
    .movie-app-logo img {
        margin-right: 10px;
    }
}
