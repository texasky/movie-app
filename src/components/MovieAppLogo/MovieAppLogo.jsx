import './MovieAppLogo.css';

const MovieAppLogo = (props) => {
    const logo = require('../../assets/movie-app-icon.jpeg');

    return (
        <div className='movie-app-logo' data-testid="movie-app-logo">
            <img src={logo} width={40} height={40} alt='MOVIE APP'/>
            <span>MOVIE APP. TYPE AND SEARCH :)</span>
        </div>
    )
}

export default MovieAppLogo;