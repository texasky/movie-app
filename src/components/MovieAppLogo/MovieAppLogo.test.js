import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import MovieAppLogo from './MovieAppLogo';

test('Movi app logo component tests', () => {
    const utils = render(<MovieAppLogo />);

    const movieAppLogo = utils.getByTestId('movie-app-logo');

    expect(movieAppLogo).toBeInTheDocument();
});

test('Movi app logo component matches snapshot', () => {
    const tree = renderer.create(<MovieAppLogo />).toJSON();
    expect(tree).toMatchSnapshot();
});