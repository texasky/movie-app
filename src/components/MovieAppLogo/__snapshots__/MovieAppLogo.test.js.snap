// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movi app logo component matches snapshot 1`] = `
<div
  className="movie-app-logo"
  data-testid="movie-app-logo"
>
  <img
    alt="MOVIE APP"
    height={40}
    src="movie-app-icon.jpeg"
    width={40}
  />
  <span>
    MOVIE APP. TYPE AND SEARCH :)
  </span>
</div>
`;
